### PR TITLE
.: move past contribex leads to emeritus <3

### PR DESCRIPTION
**Pull request by MadhavJivrajani**
Ref: https://github.com/kubernetes/community/issues/7623

### CODE REVIEW

File: prow/cmd/peribolos/OWNERS
Comment (cblecker) [inline prow/cmd/peribolos/OWNERS:10]: I'm fine remaining a peribolos approver :) (this is unrelated to the TL role)

**Review by cblecker — CHANGES_REQUESTED** [review]

File: prow/cmd/peribolos/OWNERS
Comment (MadhavJivrajani) [reply prow/cmd/peribolos/OWNERS:10]: 🎉 
Added back!

**Comment by MadhavJivrajani** [comment]
/hold 
for acks

**Review by cblecker — APPROVED** [review]

File: prow/cmd/peribolos/OWNERS
Comment (nikhita) [reply prow/cmd/peribolos/OWNERS:10]: I'd like to remain here as well! Will move myself to emeritus in a few months if I'm unable to spend time here

Comment (MadhavJivrajani) [reply prow/cmd/peribolos/OWNERS:10]: Yay!
Added back ❤️

**Review by cblecker — APPROVED** [review]

**Pull request merged**
